### PR TITLE
If charsets are not supported, include what they are in the error

### PR DESCRIPTION
**Pull request by veselov**
saves a lot on debugging

### CODE REVIEW

File: lib/iconv.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ function Iconv(fromEncoding, toEncoding)
 
   var conv = bindings.make(fixEncoding(fromEncoding),
                            fixEncoding(toEncoding));
-  if (conv === null) throw new Error('Conversion not supported.');
+  if (conv === null) throw new Error('Conversion from '+fromEncoding+' to '+toEncoding+' is not supported.');
```
Comment (bnoordhuis) [inline lib/iconv.js]: Can you wrap this at 80 columns and put spaces between the + operators?  Probably should be a `if (...) {` block now.

**Comment by bnoordhuis** [comment]
I'm fine with the change but can you add one or two regression tests?  Thanks.

**Comment by bnoordhuis** [comment]
One final nit: can you update the commit log to 50-0-72 format?  (I.e. 50 columns, blank line, one or more 72 column lines.)

**Comment by veselov** [comment]
Will address those. Can you please help me getting tests to run? I'm getting `Error: Cannot find module '../build/Debug/iconv.node'`, not sure how build the "debug" version. Thank you.

**Comment by bnoordhuis** [comment]
Run `node-gyp rebuild` from the top-level directory.

**Comment by veselov** [comment]
Thank you. I closed this PR as I don't know how to modify pushed comment logs.

**Pull request closed without merging**